cypress/fixtures/analysis.json

```diff
@@ -185,5 +185,19 @@
         "target": ["Containerization"],
         "appName": "example-1",
         "storyPoints": 1
+    },
+    "source_analysis_on_nexus_app": {
+        "source": "Source code",
+        "target": ["OpenJDK"],
```

Comment:
Let's add cloud-readiness as well as Ramon suggested.

Reply:
Solved in 212f6dcc7e204c69310579b7db2896b8c80944ed.